Code/GraphMol/MolStandardize/MolStandardize.h

```diff
@@ -39,7 +39,11 @@ namespace MolStandardize {
 */
 struct RDKIT_MOLSTANDARDIZE_EXPORT CleanupParameters {
   // TODO reveal all parameters
-  std::string rdbase = std::getenv("RDBASE");
+ private:
+  const char *rdbase_cstr = std::getenv("RDBASE");
```

Comment:
I'm thinking this it is not a good idea to execute this during a static initializer (I think that is what happens here).

I think on windows you aren't supposed to use any standard library functions when calling DllMain.

I would move this to the constructor.

Reply:
This is really no different from we’ve been doing since the last release. It hasn’t been a problem on any system

Reply:
This is really no different from we’ve been doing since the last release. It hasn’t been a problem on any system